core/src/components/radio/radio.tsx

```diff
@@ -73,6 +73,11 @@ export class Radio implements ComponentInterface {
    */
   @Event() ionSelect!: EventEmitter<CheckedInputChangeEvent>;
 
+  /**
+   * Emitted when checked radio button is selected.
```

Comment:
```suggestion
   * Emitted when checked radio button is selected.
   * @internal
```

Reply:
1) I assume this "@internal" is only a documentation thing?
2) If the ionSelect event is not "@internal", because the event could be useful to listen for by higher level code, then I guess the same could be true for the ionDeselect event. So not sure one needs to distinguish between the two events in this regard.

Reply:
The fact i want `ionDeselect` is because I would like to keep this event out of the public API, so developers do not rely on it.

you probably right, let's make both internal.